distributed/scheduler.py

```diff
@@ -2970,7 +2970,7 @@ def client_desires_keys(self, keys=None, client=None):
             if ts._state in ("memory", "erred"):
                 self.report_on_key(ts=ts, client=client)
 
-    def client_releases_keys(self, keys=None, client=None):
+    def _client_releases_keys(self, keys=None, client=None, recommendations: dict = {}):
```

Comment:
Should we be concerned about a default mutable argument like this?  Or does it not matter because everyone always provides the recommendations= parameter?

Reply:
Good point. Yeah we can make this `None`, which would be safer. Thanks!

```suggestion
    def _client_releases_keys(self, keys=None, client=None, recommendations: dict = None):
```